### ISSUE TITLE
CREATE SEQUENCE is strict about the order of its parameters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#14]
```
What steps will reproduce the problem?
CREATE SEQUENCE SEQ_GEN_SEQUENCE INCREMENT BY 50 START WITH 50

What is the expected output? What do you see instead?
The same as with:
CREATE SEQUENCE SEQ_GEN_SEQUENCE START WITH 50 INCREMENT BY 50

What version of the product are you using? On what operating system, file
system, and virtual machine?
H2
1.0.66 (2008-01-18)

Do you know a workaround?
This sql statement is generated by an O/R mapper

How important/urgent is the problem for you?
would be nice

In your view, is this a defect or a feature request?
feature request

Please provide any additional information below.


```

Original issue reported on code.google.com by `frank.sc...@gmail.com` on 11 Mar 2008 at 6:14

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Could you tell me which O/R mapper creates this statement, and what version?

Regards,
Thomas

```

Original comment by `thomas.t...@gmail.com` on 11 Mar 2008 at 7:42
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
It is toplink essentials v2.1-b24 using toplink.target-database=Oracle, see
oracle.toplink.essentials.platform.database.oracle.OraclePlatform#buildSequenceO
bjectCreationWriter(...)

Kind regards,
Frank
```

Original comment by `frank.sc...@gmail.com` on 11 Mar 2008 at 8:37

**Comment by GoogleCodeExporter** [comment]
```
I suggest to use the H2 platform ('target-database').
If this class is not included in the O/R mapper version you use,
the platform is also included in the H2 source code:
src/tools/oracle/toplink/essentials/platform/database/H2Platform.java.txt
If you don't want to compile this class yourself, I suggest
to use the HSQLDB or PostgreSQL platform, not Oracle. 
```

Original comment by `thomas.t...@gmail.com` on 15 Mar 2008 at 11:28
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Let's say H2 is 'strict'.
```

Original comment by `thomas.t...@gmail.com` on 15 Mar 2008 at 11:29
- Changed title: **CREATE SEQUENCE is strict about the order of its parameters**

**Comment by GoogleCodeExporter** [comment]
```
This will be supported in the next release.
```

Original comment by `thomas.t...@gmail.com` on 23 Mar 2008 at 5:22
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.0.69 (2008-03-29)
```

Original comment by `thomas.t...@gmail.com` on 29 Mar 2008 at 12:30
- Changed state: **Fixed**
